repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -2029,6 +2126,10 @@ export const getProductGroupByNocAndId = async (noc: string, id: number): Promis
     try {
         const result = await executeQuery<GroupOfProductsDb[]>(query, [noc, id]);
 
+        if (result.length > 1) {
```

Comment:
We dont want to do this, as then it will return undefined if there is more than 1 result.

Reply:
return undefined;

Reply:
??

Reply:
corrected the function

Reply:
I dont think thats correct either - i would just remove this check. It's not doing anything.